lib/mjml/parser.rb

```diff
@@ -29,20 +26,21 @@ def call!(template)
     private
 
     def exec!
-      MJML.logger.debug("Template:\n #{@template}")
       raise InvalidTemplate if @template.empty?
 
-      MJML.logger.debug("Partial: #{partial?}")
       return @template if partial?
 
       out, err = should_get_outpout_from_file? ? output_from_file : output_from_memory
       parsed = parse_output(out)
 
-      MJML.logger.debug("Output:\n #{parsed[:output]}")
```

Comment:
Not sure about logging. Could you explain please while it is not helpful? As consumer you always can silence them by changing logger level. As maintainer it is helpful to have them during the debug process.

Reply:
Actually a couple of reasons:
 * the main one is that mjml works more like a transpiler rather than a generator, the code it generates is arguably readable and it ends up either way in your rendered template (either mailer preview or the page that was rendered, which ultimately are superior to inspect). In a nutshell, if haml doesn't do it, why should we do it?!
 * if the templates are parsed properly, nobody cares what's the output, if there's an error, the changes I added related to the error messages clearly state where the issues are.
 * there's no way to disable this in dev/test environment, and you end up with gigabytes of log data, not to mention it that it makes important logs messages harder to find (a template generates hundreds of lines of output in the log, to the point that the main request log message is lost and requires extra effort to scroll around and find it).

Long story short, we either separate the debugging from the main logger, or we remove it because it harms more than it helps now.

Let me know what makes you want to have it like it is now?

Reply:
Interesting. Default Logger level is error so debug messages shouldn't appear at all unless you specify your custom logger with another logger level. https://github.com/m-basov/mjml-ruby/blob/205aa2f9d9f0a58e0a4f390605a2aac824e20fde/lib/mjml/logger.rb#L8

Reply:
I don't have very strong opinion on this so if my comment above doesn't work feel free to merge.